modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -56,9 +52,6 @@
 		flex-direction: var( --n-tabs-direction );
 		gap: var( --n-tabs-gap );
 		text-align: $start;
-		height: var( --n-tabs-height );
-		overflow-x: var( --n-tabs-overflow-x );
```

Comment:
Do we want to allow the option of horizontal overflow?

Reply:
These variables weren't used anymore.
The were used in the original version, but I forgot to remove them when we cleaned up the widget.

Reply:
@davseve 

The original idea was to allow for vertical scrolling, in some cases.